com.unity.shadergraph/Documentation~/Sample-Virtual-Texture-Node.md

```diff
@@ -33,7 +33,8 @@ The Sample Virtual Texture node has several settings available for you to specif
 |:------------ |:-------------|:-----|:---|
 |  Lod Mode   | Dropdown | Automatic, Lod Level, Lod Bias, Derivatives | Sets the specific Lod mode to use when sampling the textures. |
 |  Quality   | Dropdown | Low, High | Sets the quality mode to use when sampling the textures.  |
-|  Manual Streaming  | Toggle | Enabled/Disabled | Determines whether the node should use automatic streaming or manual streaming. |
+|  Automatic Streaming  | Toggle | Enabled/Disabled | Determines whether the node should use automatic streaming or manual streaming. |
+|  Disable Global Mip Bias   | Toggle | Enabled/Disabled |  Disables the automatic global mip bias imposed by the runtime. This bias is set during certain dynamic resolution scaling algorithms to improve detail reconstruction. |
```

Comment:
Remove passive voice: 

"Disables the global mipmap bias that Unity automatically imposes at runtime. Unity sets this bias during certain dynamic resolution scaling algorithms to improve detail reconstruction."

Reply:
Fixed! thank you